Language/Variables/Conversion/floatCast.adoc

```diff
@@ -48,7 +48,7 @@ Data type: `float`.
 
 [float]
 === Notes and Warnings
-See the reference for link:../../data-types/float[float] for details about the precision and limitations of floating point numbers on Arduino.
+See the reference for link:../../data-types/float[`float`] for details about the precision and limitations of floating point numbers on Arduino.
```

Comment:
Here we want the link to be colored because we are specifically directing the reader to that page so we want it to be obvious this is a link.

My idea of only coloring the first occurrence of a link is only a general guideline. There will be cases where it makes sense to have the same link colored multiple times on the same page. It's really just about doing what's best for the reader. Having one colored link at line 20 and another at line 51 is certainly not going to cause the undesirable "sea of teal" effect.
```suggestion
See the reference for `link:../../data-types/float[float]` for details about the precision and limitations of floating point numbers on Arduino.
```

Reply:
Thanks a lot for the feedback. Sure, I'll keep this in mind while making new changes.